core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.spec.ts

```diff
@@ -140,14 +141,10 @@ describe('EmaContentletToolsComponent', () => {
                     } as ActionPayload);
                 });
 
-                it('should call addForm on Form option click', () => {
-                    const addSpy = jest.spyOn(spectator.component.addForm, 'emit');
-                    spectator.click('[data-testId="add-top-button"]');
-                    spectator.click(byText('Form'));
-                    expect(addSpy).toHaveBeenCalledWith({
-                        ...contentletAreaMock.payload,
-                        position: 'before'
-                    } as ActionPayload);
+                it('should not call addForm on Form option click', () => {
+                    spectator.click('[data-testId="add-bottom-button"]');
```

Comment:
@rjvelazco When we use the data-testId pattern, I suggest using the `byTestId` selector from Spectator as a standard, like this:

```ts
import { byTestId } from '@ngneat/spectator';

spectator.click(byTestId('add-bottom-button'));
```

This way we avoid typos.

Reply:
Done!